cookiebot.php

```diff
@@ -1583,6 +1583,23 @@ function debug_page() {
 		$active_plugins = get_option( 'active_plugins' );
 
 
+		//Dashboard missing login info
```

Comment:
@lordlukas10  Consider moving this to a helper function that returns the dashboard config string